client/src/modules/stock/lots/modals/search.modal.html

```diff
@@ -11,58 +11,83 @@
     </ol>
   </div>
 
-  <div class="modal-body">
+  <div class="modal-body" style="overflow: auto; max-height: 600px; padding: 0px">
+    <uib-tabset>
+      <uib-tab index="0" heading="{{ 'FORM.LABELS.SEARCH_QUERRIES' | translate}}" data-custom-filter-tab>
+        <div class="tab-body">
+          <!-- depot -->          
+          <bh-depot-select
+            depot-id="$ctrl.searchQueries.depot_uuid"
+            name="depot_id"
+            on-select-callback="$ctrl.onSelectDepot(depot)">
+            <bh-clear on-clear="$ctrl.clear('depot_uuid')"></bh-clear>
+          </bh-depot-select>
 
-    <!-- depot -->
-    <div class="form-group">
-      <label class="control-label" translate>STOCK.DEPOT</label>
+          <!-- Inventaire -->
+          <bh-inventory-select
+            inventory-id="$ctrl.searchQueries.inventory_uuid"
```

Comment:
Could you rename this parameter `inventory-uuid`?  Since there isn't actually an inventory ID column, it will save confusion in the future.

Note that the `name` here should also be updated.